apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -879,12 +892,20 @@ export class LedgerComponent implements OnInit, OnDestroy {
-
+                let checkedEntriesName: any[];
+                if (this.ledgerView === LedgerViewEnum.TView) {
+                    const debitTransactions = lt.debitTransactions ?? [];
+                    const creditTransactions = lt.creditTransactions ?? [];
+                    checkedEntriesName = uniq([
+                      ...debitTransactions.filter(debitTransaction => debitTransaction.isChecked).map(debitTransaction => ({ uniqueName: debitTransaction.entryUniqueName, type: 'debit' })),
+                      ...creditTransactions.filter(creditTransaction => creditTransaction.isChecked).map(creditTransaction => ({ uniqueName: creditTransaction.entryUniqueName, type: 'credit' })),
+                    ]);
+                } else {
+                    checkedEntriesName = uniq([
+                        ...lt?.debitCreditTransactions?.filter(f => f.isChecked).map(dt => ({ uniqueName: dt.entryUniqueName, type: dt.type }))
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)